backend/app/views/spree/admin/shared/_sidebar_left.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="sidebar">
+  <%= render partial: 'spree/admin/shared/sidebar_left_header' %>
```

Comment:
Should these be `sidebar_left_#{thing}`? Seems like this is the one and only main sidebar. Until we're planning on having a right sidebar I'd like to kill the `left`.